include/verilated_vcd_c.cpp

```diff
@@ -881,23 +1211,23 @@ double doub = 0;
 void vcdInit(VerilatedVcd* vcdp, void* userthis, vluint32_t code) {
     vcdp->scopeEscape('.');
     vcdp->module("top");
-     vcdp->declBus(0x2, "v1",-1,5,1);
-     vcdp->declBus(0x3, "v2",-1,6,0);
```

Comment:
Please revert the formatting of the rest of the file and add the clang-format off/on's.